jdaviz/configs/imviz/wcs_utils.py

```diff
@@ -575,7 +576,13 @@ def compute_scale(wcs, fiducial, disp_axis, pscale_ratio=1):
     if spectral and disp_axis is None:  # pragma: no cover
         raise ValueError('If input WCS is spectral, a disp_axis must be given')
 
-    crpix = np.array(wcs.invert(*fiducial))
+    if wcs.in_image(*fiducial):

```

Comment:
@bmorris3 should review this one. And also since he mentioned that he lifted this code from `jwst`, I wonder if a similar patch has to go to `jwst` also.